lib/suma/commerce/checkout.rb

```diff
@@ -113,7 +113,8 @@ def requires_payment_instrument?
     return self.chargeable_amount?
   end
 
-  def checkout_prohibited_reason
+  def checkout_prohibited_reason(now: Time.now)
```

Comment:
Get rid of the default

Reply:
As a rule, don't default to an external dependency like `Time.now` in the models unless it makes something much much simpler.

Reply:
@rgalanakis For some reason `Grape::Middleware::Formatter` is complaining. I tried to diagnose this but I'm not sure what's happening. Errors when removing a default value:

![Screenshot 2023-12-15 at 9 50 34 AM](https://github.com/lithictech/suma/assets/66847768/8adebb71-9b8c-46a9-b660-8e06edd26a6e)